client-misk/src/main/kotlin/app/cash/backfila/client/misk/client/BackfilaMiskClientModule.kt

```diff
@@ -16,7 +16,7 @@ import retrofit2.converter.wire.WireConverterFactory
 
 /**
  * Use this to connect to a real Backfila service in staging or production. You will also need to
- * install a [BackfillModule].
+ * install a [MiskBackfillModule].
```

Comment:
This sounds like a breaking change. Do we need a deprecation/typealais for the old BackfillModule?

Reply:
Ok I think we need deprecated type aliases: seems like `BackfillModule` is installed directly by a bunch of our internal Misk apps.